supported_formats_and_operations.md

```diff
@@ -40,8 +40,8 @@ The supported preprocessing operations.
   - `kwargs`
     - `mode` can be one of `per_sample` (percentiles are computed for each sample individually), `per_dataset` (percentiles are computed for the entire dataset)
     - `axes` the subset of axes to normalize jointly. For example `xy` to normalize the two image axes for 2d data jointly. The batch axis (`b`) is not valid here.
-    - `min_percentile` the lower percentile used for normalization, in range 0 to 100.
-    - `max_percentile` the upper percentile used for normalization, in range 0 to 100. Has to be bigger than `upper_percentile`.
+    - `min_percentile` the lower percentile used for normalization, in range 0 to 1
+    - `max_percentile` the upper percentile used for normalization, in range 0 to 1. Has to be bigger than `upper_percentile`.
```

Comment:
percentile usually refers to a range from 0 to 100, not 0 to 1.
See https://en.wikipedia.org/wiki/Percentile.
I think it would be confusing if we change this from the common definition.
@frauzufall how do you handle this in `CSBDeep`?

Reply:
Yes, this was a suggestion coming from [#43](https://github.com/bioimage-io/configuration/pull/43#issuecomment-733554840). On one side I agree that percentiles are quantiles, but on the other, it does really depend on the software you use, and as @FynnBe suggested, some cases can be ambiguous.

Reply:
No, @FynnBe did not suggest to change the value range to `[0, 1]`. Instead he suggested to enforce `max_percentile > 1` in order to avoid confusion about the value ranges.